### PR TITLE
Remove material, redesign

### PR DESCRIPTION
**Pull request by Flightan**

### CODE REVIEW

**Comment by tmerlier** [comment]
Error when load a harvest detail page.
http://localhost:3000/catalogs/5387a33e785588482f5acb44/harvest/5806b79f3cd7fa3b5a017911

**Comment by tmerlier** [comment]
The display of percentages is very small compare to charts.

tiny-stats

**Comment by tmerlier** [comment]
When the data is loaded, the div is no longer stuck to the footer.
data1

## after

data0

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -23,65 +72,34 @@ class Home extends Component {
...
+          <Link style={styles.catalogLinks} to="catalogs">Explore the 106 catalogs</Link>
+        </div>
 
-          <Paper style={styles.paper} zDepth={2}>
-            <h3>Not Yet Published</h3>
-            {notPublishedYet}
-          </Paper>
+        <div style={styles.datagouv}>
+          <p style={styles.datagouvIntro}>
+            Votre organisation gère des <b>données géographiques</b> avec des outils compatibles Inspire et souhaite les rendre disponibles sans effort sur <a style={styles.datagouvLink} href="http://www.data.gouv.fr/fr/">data.gouv.fr</a>
```
Comment (tmerlier) [inline src/components/Home/Home.js:89]: It should to be in english.

File: src/components/Section/HarvestsSection/HarvestDetail/HarvestDetail.js [hunk 2]
```diff
@@ -98,16 +96,16 @@ class HarvestDetail extends Component {
...
                 <h1 style={styles.h1} className="ui header">{this.state.catalog.name}</h1>
               </Link>
-              <Subheader>Harvest ID: {this.state.harvest._id}</Subheader>
+              <h2>Harvest ID: {this.state.harvest._id}</h2>
             </div>
 
-            <Chip
+            <span
               style={styles.chip}
               labelColor={this.context.muiTheme.palette.alternateTextColor}
```
Comment (tmerlier) [inline src/components/Section/HarvestsSection/HarvestDetail/HarvestDetail.js]: Error when load a harvest detail page.

```
warning.js:36 Warning: Failed context type: Required context `muiTheme` was not specified in `HarvestDetail`.
    in HarvestDetail (created by RouterContext)
    in div (at App.js:28)
    in div (at App.js:26)
    in App (created by RouterContext)
    in RouterContext (created by Router)
    in Router (at index.js:20)printWarning @ warning.js:36
HarvestDetail.js:101 Uncaught (in promise) TypeError: Cannot read property 'palette' of undefined(…)
```

[HERE](http://localhost:3000/catalogs/5387a33e785588482f5acb44/harvest/5806b79f3cd7fa3b5a017911)

File: src/components/Section/Section.js [hunk 3]
```diff
@@ -16,7 +12,7 @@ class Section extends Component {
   }
 
   render() {
-    const expandIcon = this.state.open ? <NavigationExpandLessIcon /> : <NavigationExpandMoreIcon />
+    const expandIcon = this.state.open ?  <span>v</span> : <span>></span>
```
Comment (tmerlier) [inline src/components/Section/Section.js]: Windows 98 nostalgia?

**Review by tmerlier — DISMISSED** [review]

**Comment by jdesboeufs** [comment]
@tmerlier ?

**Comment by tmerlier** [comment]
When only one data is sent to a Doughnut Chart he's change in a percentage. It would also change its size as for now, it is smaller than other.
stats

**Comment by Flightan** [comment]
@tmerlier ?

**Comment by Flightan** [comment]
@tmerlier ?

**Comment by tmerlier** [comment]
## GTM

**Pull request merged**